kibana/fleet.go

```diff
@@ -493,3 +508,202 @@ func (client *Client) GetFleetServerHost(request GetFleetServerHostRequest) (*Ge
 
 	return &resp.Item, nil
 }
+
+//
+// Fleet Package Policy
+//
+
+// https://www.elastic.co/guide/en/fleet/8.8/fleet-apis.html#createPackagePolicy
+// request https://www.elastic.co/guide/en/fleet/8.8/fleet-apis.html#package_policy_request
+type PackagePolicyRequest struct {
```

Comment:
Thanks! Is https://github.com/elastic/elastic-agent/blob/main/pkg/testing/tools/fleet_package_policy_api.go getting removed when this merges?

Reply:
Yeah, saw the "TODO" there and addressed along with other new additions that were needed for agent tampering test cases. Will pull these from the library once it merges and update agent E2E tests.